Cargo.toml

```diff
@@ -1,4 +1,5 @@
 [workspace]
+resolver = "2"
```

Comment:
A ton of new packages were added to `Cargo.lock`. I am curious if that is mostly because of the resolver change or because of the switch to `deno_runtime`.

Reply:
It's because of the change to `deno_runtime`. The resolver change is just necessary to make things build.